recipes/bioconductor-biocgenerics/meta.yaml

```diff
@@ -22,9 +22,9 @@ build:
 # Suggests: Biobase, S4Vectors, IRanges, GenomicRanges, DelayedArray, Biostrings, Rsamtools, AnnotationDbi, affy, affyPLM, DESeq2, flowClust, MSnbase, annotate, RUnit
 requirements:
   host:
-    - r-base
+    - r-base >=3.6.3
```

Comment:
I think this is not the right way to do it. You will need to make such change to each and every recipe that uses `r-base` and needs aarch64 support.

IMO the better way is to pin the version at https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml. It will have to be added around here - https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/e433c4cada823b343e0ba0d31dc68a83a9e6543e/recipe/conda_build_config.yaml#L800 and most probably will need a selector like `# [linux and aarch64]`

Reply:
Am I right to modify it like this?  Although it specified version 4.2 by `- 4.2   # [not win]`, it still uses version 3.5 when I use `conda build .` in the arm server. I don't understand.
<img width="1291" alt="image" src="https://github.com/bioconda/bioconda-recipes/assets/37299202/c70ea180-376a-4ba0-ba9d-eb946d1d2723">

Reply:
I guess you made this change in your local clone of `conda-forge-pinning-feedstock` ? This won't work!
The tools use the released config. I.e. once you open a PR at conda-forge-pinning-feedstock and it is merged then the tools will use the new config.
Until then you will need to modify the recipe/meta.yaml as you did in your initial version of this PR.

Reply:
I just added a comment at https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/5750/files#r1560772006

Could you please explain what exactly is the problem you face here ?
I believe the builds should use R-base 4.x (Bioconductor 3.18+ needs 4.3).
Why do you want 3.6.3 ?

Reply:
I used `conda build .` to build _bioconductor-BiocGenerics_ , but it told me that r-base=3.5 can't be found.  And then I used `conda search r-base` to query the supported version of arm and found that the minimum version of r-base was **3.6.3**. So I fix it by **r-base >=3.6.3**

Reply:
If `conda-forge-pinning-feedstock` is already marked as a minimum version of 4.2, why would it use version  r-base=3.5 when I was building it?

Reply:
I am not sure why it uses 3.5 locally.
It uses 4.3.3 at Azure for the Linux x86_64 build - https://dev.azure.com/bioconda/25074ded-3133-43ca-af80-895ab87b53cb/_apis/build/builds/55921/logs/26
```
+ r-base                            4.3.3  hb8ee39d_0           conda-forge/linux-64      26MB
```

Same for Linux ARM64 for my PR:
- https://github.com/bioconda/bioconda-recipes/pull/46177
- https://circleci.com/api/v1.1/project/github/bioconda/bioconda-recipes/182712/output/107/0?file=true&allocation-id=660149c5e42ee110e359de92-0-build%2FABCDEFGH